src/embeddings/model_loader.py

```
@@ -0,0 +1,184 @@
+            True if models are compatible
+            
+        Raises:
+            ModelLoadError: If models are incompatible
+        """
+        try:
+            # Test text model
+            text_embedding = text_model.encode("test", convert_to_numpy=True)
+            text_dim = text_embedding.shape[0]
+            
+            # Test CLIP model with dummy image
+            dummy_image = Image.new('RGB', (224, 224), color='white')
```

Ruff (0.13.1)

Local variable `dummy_image` is assigned to but never used

Remove assignment to unused variable `dummy_image`

(F841)